wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift

```diff
@@ -33,12 +33,13 @@ final class SelfProfileViewController: UIViewController {
     // MARK: - Views
 
     private let settingsController: SettingsTableViewController
-    private let accountSelectorController = AccountSelectorController()
+    private weak var accountSelectorView: AccountSelectorView?
     private let profileContainerView = UIView()
     private let profileHeaderViewController: ProfileHeaderViewController
     private let profileImagePicker = ProfileImagePickerManager()
 
     let userSession: UserSession
+    private let accountSelector: AccountSelector?
```

Comment:
Not sure if we should have this property inside `AccountSelectorView` instead of having a `AccountSelectorViewDelegate`. Opinions?